src/screening/ScreeningQueue.jsx

```diff
@@ -34,22 +33,17 @@ let ScreeningQueue = props => {
     <div>
       <CardList
         card={PatientCard}
-        /*
-        * Calling this cardListFilters till we decide 
-        * what happens to the already existing filter.
-        */
-        cardListFilters={ScreeningFilters}
         delayInterval={0}
         dispatch={props.dispatch}
         fetchListActionCreator={fetchListActionCreator}
         filters={[...props.filters, patientObjByEncounterTypeFilter(ENCOUNTER_TYPES.CheckInEncounterType.uuid, 'include')]}
         getPatientIdentifiers={utils.getPatientIdentifiers}
         loading={props.updating}
         onMountOtherActionCreators={onMountOtherActionCreators}
-        optionalFilters={PATIENT_IDENTIFIER_FILTERS}
-        optionalFiltersType='or'
         rowData={props.rowData}
         rowSelectedActionCreators={[patientActions.setSelectedPatient, ...props.rowSelectedActionCreators]}
+        ScreeningFilters={ScreeningFilters}
+        searchFilterFields={['name.givenName', 'name.familyName', 'identifiers.0.identifier', 'identifiers.1.identifier', 'identifiers.2.identifier']}
```

Comment:
A patient can have 0 to n identifiers... is there are way search matcher can handle this?  If not, we can go with this pattern for now, though let's at 'identifiers.4..."  to "identifiers.8...." just to be safe